src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -30,1327 +31,990 @@
+	public static final RegistryObject<BlockItem> CHIPBOARD = registerBlockItem(MSBlocks.CHIPBOARD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> WOOD_SHAVINGS = registerBlockItem(MSBlocks.WOOD_SHAVINGS, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> DENSE_CLOUD = registerBlockItem(MSBlocks.DENSE_CLOUD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> BRIGHT_DENSE_CLOUD = registerBlockItem(MSBlocks.BRIGHT_DENSE_CLOUD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> SUGAR_CUBE = registerBlockItem(MSBlocks.SUGAR_CUBE, MSItemGroup.LANDS);
+	
+	
+	
+	//Land Tree Blocks
+	public static final RegistryObject<BlockItem> GLOWING_LOG = registerBlockItem(MSBlocks.GLOWING_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> FROST_LOG = registerBlockItem(MSBlocks.FROST_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> RAINBOW_LOG = registerBlockItem(MSBlocks.RAINBOW_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> END_LOG = registerBlockItem(MSBlocks.END_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> VINE_LOG = registerBlockItem(MSBlocks.VINE_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> FLOWERY_VINE_LOG = registerBlockItem(MSBlocks.FLOWERY_VINE_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> DEAD_LOG = registerBlockItem(MSBlocks.DEAD_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> PETRIFIED_LOG = registerBlockItem(MSBlocks.PETRIFIED_LOG, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> GLOWING_WOOD = registerBlockItem(MSBlocks.GLOWING_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> FROST_WOOD = registerBlockItem(MSBlocks.FROST_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> RAINBOW_WOOD = registerBlockItem(MSBlocks.RAINBOW_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> END_WOOD = registerBlockItem(MSBlocks.END_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> VINE_WOOD = registerBlockItem(MSBlocks.VINE_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> FLOWERY_VINE_WOOD = registerBlockItem(MSBlocks.FLOWERY_VINE_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> DEAD_WOOD = registerBlockItem(MSBlocks.DEAD_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> PETRIFIED_WOOD = registerBlockItem(MSBlocks.PETRIFIED_WOOD, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> GLOWING_PLANKS = registerBlockItem(MSBlocks.GLOWING_PLANKS, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> FROST_PLANKS = registerBlockItem(MSBlocks.FROST_PLANKS, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> RAINBOW_PLANKS = registerBlockItem(MSBlocks.RAINBOW_PLANKS, MSItemGr
```

Comment:
`getRegistryName()` is not called here, so the comment does not make much sense.

Reply:
Understood, I wanted to add a comment to explain why we were letting it go without being checked. Would the following make sense to you?
`//assumed location() will occur without fail as it works through DeferredRegistry`

Reply:
Not really, since `location()` should give a non-null result regardless of if it came from the deferred registry or not.
However, `getKey()` has a nullable annotation, while we assume that it is not null.

Reply:
Is what I put now acceptable?

Reply:
idk i can't read i'm too drunk